src/content/docs/journey-migration/migrating-apm.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: "Install APM agents"
+metaDescription: "How to setup APM monitoring agents in New Relic."
+---
+
+import journeyAPMOverview from 'images/apm_screenshot-crop_overview.png'
+
+import journeyAPMPrefabAlerts from 'images/apm_screenshot-crop_precon-alerts.png'
+
+import journeyAPMDash from 'images/apm_screenshot-crop_journey-dash.png'
+
+import journeyAPMDash2 from 'images/apm_screenshot-crop_journey-dash-two.png'
+
+While setting up infrastructure monitoring provides a great foundation for understanding your systems, the best way to truly know your data comes from monitoring application performance. New Relic APM provides extensive, unified monitoring for all your applications. 
```

Comment:
Is it the best way to "truly know" your _data_, or perhaps it's something more important like your customer experience, your system etc. Data is a means to an end.

Reply:
Fixed!